src/backend/distributed/commands/multi_copy.c

```diff
@@ -2346,6 +2346,8 @@ CitusCopyDestReceiverStartup(DestReceiver *dest, int operation,
 
 	copyDest->shardStateHash = CreateShardStateHash(TopTransactionContext);
 	copyDest->connectionStateHash = CreateConnectionStateHash(TopTransactionContext);
+
+	RecordRelationAccessIfReferenceTable(tableId, PLACEMENT_ACCESS_DML);
```

Comment:
Do we also need to set the connection type to sequential if this is a reference table? Maybe that can be a separate PR with its own tests though.

Reply:
It seems that we

> Do we also need to set the connection type to sequential if this is a reference table? Maybe that can be a separate PR with its own tests though.

Never mind, calling `RecordRelationAccessIfReferenceTable()` does the switch.